app-header-layout/app-header-layout.html

```diff
@@ -55,6 +55,13 @@
     <style>
       :host {
         display: block;
+
+        /**
+         * Force app-header-layout to have its own stacking context so that its parent can
+         * control the stacking of it relative to other elements (e.g. app-drawer-layout).
+         */
+        position: relative;
```

Comment:
Since we have `position: relative` set on the host now, we can remove the same in `:host([has-scrolling-region])` https://github.com/PolymerLabs/app-layout/blob/app-header-layout/app-header-layout/app-header-layout.html#L68